scipy/optimize/_highs/src/simplex/SimplexConst.h

```diff
@@ -103,8 +103,7 @@ enum SimplexPriceStrategy {
   SIMPLEX_PRICE_STRATEGY_ROW,
   SIMPLEX_PRICE_STRATEGY_ROW_SWITCH,
   SIMPLEX_PRICE_STRATEGY_ROW_SWITCH_COL_SWITCH,
-  SIMPLEX_PRICE_STRATEGY_ROW_ULTRA,
-  SIMPLEX_PRICE_STRATEGY_MAX = SIMPLEX_PRICE_STRATEGY_ROW_ULTRA
+  SIMPLEX_PRICE_STRATEGY_MAX = SIMPLEX_PRICE_STRATEGY_ROW_SWITCH_COL_SWITCH
```

Comment:
We're not exposing this option though. I'm wondering whether we should cut back on which options we expose. It would be better to never offer experimental options than to remove them later.

Reply:
Now that I think about it, let's have a think about which options we don't want to expose via `_linprog_highs.py`.  I think I'm fine with `highs_wrapper` letting you call any option that HiGHS will accept, but some bumpers would be good to disallow experimental options at the Python layer.  I think a good place to have the discussion about what we consider experimental in https://github.com/scipy/scipy/pull/12043

Proposed: Instead of checking all options in `_linprog_highs.py`, we could check only for experimental options and either warn or prevent users from using those.  That would mean that this PR is good as is and we can add more checks as we come to a consensus on what is experimental for the purposes of scipy

Reply:
I only remember `SIMPLEX_STRATEGY_PRIMAL` being noted as experimental, do you remember Julian mentioning that any of the `SimplexPriceStrategies` were experimental?

Reply:
`SIMPLEX_STRATEGY_PRIMAL` is now removed